src/components/Search/SearchAutocompleteInput.tsx

```diff
@@ -231,27 +232,50 @@ function SearchAutocompleteInput(
                         onBlur={() => {
                             autocompleteListRef?.current?.updateExternalTextInputFocus(false);
                             focusedSharedValue.set(false);
+
                             onBlur?.();
                         }}
-                        isLoading={isSearchingForReports}
-                        ref={ref}
+                        ref={(ref: BaseTextInputRef) => {
+                            if (typeof forwardedRef === 'function') {
+                                forwardedRef(ref);
+                            } else if (forwardedRef && 'current' in forwardedRef) {
+                                // eslint-disable-next-line no-param-reassign
+                                forwardedRef.current = ref;
+                            }
+                            textInput.current = ref;
+                        }}
                         type="markdown"
                         multiline={false}
                         parser={parser}
                         selection={selection}
                     />
                 </View>
-                {!!value && (
-                    <Animated.View
-                        style={styles.pr3}
-                        layout={SearchFiltersButtonTransition}
-                    >
+                <Animated.View
```

Comment:
```suggestion
                {!!value && (
                    <Animated.View
```
We can remove multiple usages of isSearchingForReports for the loader. Let's render the entire part if the value is available.

Reply:
I mean let's keep how it was earlier, where the clear button renders with `{!!value && (` condition.